src/app/components/FilterPopup/FilterPopup.jsx

```diff
@@ -237,7 +238,11 @@ class FilterPopup extends React.Component {
                   onFilterClick={this.onFilterClick}
                 />
 
-                <FieldsetDate />
+                <FieldsetDate
+                  legend="Date"
+                  filterId="date"
+                  selectedFacets={this.state.selectedFilters}
```

Comment:
Check line 161, `selectedFilters` is already a variable so you shouldn't do `this.state.selectedFilters`.